app/main.js

```diff
@@ -54,6 +56,34 @@ function createWindow() {
 
   global.mainWindowId = win.id;
 
+  let appIcon = null;
+  app.whenReady().then(() => {
+    const iconExists = fs.existsSync(iconPath);
+    appIcon = new Tray(iconExists ? iconPath : './build/icon.ico');
+    const contextMenu = Menu.buildFromTemplate([
+      {
+        label: 'Show',
+        click: function() {
+          global.win.show();
+          global.win.maximize();
```

Comment:
Save the state of the window before minimizing it, so it can be restored in the correct state (minimized vs. maximized). Currently you always maximize the window, even when the windows wasn't previously maximized